src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/services9.yml

```diff
@@ -76,6 +76,10 @@ services:
         class: stdClass
         decorates: decorated
         decoration_inner_name: decorated.pif-pouf
+    deprecated_service:
+        class: stdClass
+        deprecated: true
```

Comment:
@fabpot I changed the key then, and only allow a `deprecated` key in a YAML service configuration file. If it is present, then the service is marked as deprecated (and so are services decorating or extending this service).

But I was wondering if the naming was appropriate for XML and YAML. Wouldn't be `deprecation` better then ?